app/models/form/sales/questions/savings.rb

```diff
@@ -1,8 +1,8 @@
 class Form::Sales::Questions::Savings < ::Form::Question
-  def initialize(id, hsh, page)
-    super
+  def initialize(id, hsh, page, joint_purchase:)
+    super(id, hsh, page)
     @id = "savings"
-    @check_answer_label = "Buyer’s total savings before any deposit paid"
+    @check_answer_label = "#{joint_purchase ? 'Buyers’' : 'Buyer’s'} total savings before any deposit paid"
```

Comment:
there's some inconsistency of `’` vs `'` in this PR (though I accept there is throughout the codebase too) - maybe a tech debt ticket to sort this out later as it's such a non-issue?

Reply:
(looks like we're using the first one in "buyer's" and the second one in "weren't" for example)

Reply:
Fixed it for weren't/wasn't, good spot! Don't think this PR updated buyers ones!